cmd/forwarder/config.yaml

```diff
@@ -0,0 +1,113 @@
+# Copyright 2022 Stichting ThingsIX Foundation
```

Comment:
Does this replace `cmd/forwarder/example-config.yaml`?

For the initial data-only launch we don't need a gateway registry. Please remove `gateway.gateway_registry` and `gateway.interval`.

Since the `on_chain` option is disabled and routers are fetched through the ThingsIX API you can also remove the blockchain config block since that isn't required. Currently there is a check that `blockchain.polygon` must be configured. That comes from time that there was not an API option and the only way to fetch routers was from the smart contract. This check must be made smarter that it only requires blockchain configuration if the gateway registry address is filled in and/or the `routers.on_chain` option is set.

Reply:
This was my test config. I removed it.